scalableminds/webknossos
# Proxy some URIs for Demo Instance

### Steps to test:
- if isDemoInstance is true, some urls should be proxied
- root should only be proxied if the user is not logged in, otherwise the normal wk main page should load

### Issues:
- fixes #5075 

------
